docs/sources/tempo/setup/operator/api-docs.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: TempoStack documentation
-menuTitle: TempoStack documentation
-description: TempoStack CRD and Operator configuration
+title: Operator configuration and CRD specifications
+menuTitle: API documentation
```

Comment:
I think `Operator configuration and CRD specifications` is too long for the menu. I'm open for better suggestions though :)

Reply:
```suggestion
menuTitle: Operator API documentation
```

Reply:
I've been thinking about this a bit.

What about calling the page Reference or something like that?

Reply:
I like "Reference", I've renamed it now.
Yes, the page links to auto-generated reference files.